Directory.Packages.props

```diff
@@ -182,18 +182,16 @@
     <PackageVersion Include="Microsoft.Azure.Functions.Worker.Extensions.Storage.Blobs" Version="6.6.0" />
     <PackageVersion Include="Microsoft.Azure.Functions.Worker.Extensions.Storage.Queues" Version="5.5.0" />
     <PackageVersion Include="Microsoft.Azure.Functions.Worker.Extensions.ServiceBus" Version="5.22.0" />
-
-
-
     <PackageVersion Include="Microsoft.Azure.Functions.Worker.Extensions.EventHubs" Version="6.3.6"/>
-
     <PackageVersion Include="Microsoft.Azure.Functions.Worker.OpenTelemetry" Version="1.0.0-preview1" />
     <PackageVersion Include="Microsoft.Azure.Functions.Worker.Sdk" Version="1.17.4" />
     <PackageVersion Include="Microsoft.ApplicationInsights.WorkerService" Version="2.22.0" />
-    <!-- Pinned version for Component Governance - Remove when root dependencies are updated -->
+
+    <!-- Pinned versions for Component Governance - Remove when root dependencies are updated -->
     <PackageVersion Include="Azure.Core" Version="1.43.0" />
     <PackageVersion Include="Azure.Identity" Version="1.12.0" />
-    <PackageVersion Include="Newtonsoft.Json" Version="13.0.3" />
     <!-- https://github.com/Azure/azure-cosmos-dotnet-v3/pull/3313 -->
+    <PackageVersion Include="Newtonsoft.Json" Version="13.0.3" />
```

Comment:
Is the comment above still accurate?

Reply:
Yes - that is the reason why we need to "lift" Newtonsoft.Json - because of CosmosDB.